presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -59,6 +59,7 @@
 import static java.lang.Float.floatToRawIntBits;
 import static java.lang.Float.intBitsToFloat;
 import static java.lang.String.format;
+import org.apache.commons.math3.special.Erf;
```

Comment:
This import is in the wrong place, which will cause the build to fail with a Checkstyle error. It needs to be in the first section in sorted order. If you use our code style for IntelliJ, it will automatically handle the imports: https://github.com/airlift/codestyle